managed/ui/src/components/universes/DeleteUniverse/DeleteUniverse.js

```diff
@@ -62,7 +67,7 @@ export default class DeleteUniverse extends Component {
     } = this.props;
     this.props.onHide();
     if (type === 'primary') {
-      this.props.submitDeleteUniverse(data.universeUUID, this.state.isForceDelete);
+      this.props.submitDeleteUniverse(data.universeUUID, this.state.isForceDelete, this.state.isDeleteBackups);
```

Comment:
Destruct submitDeleteUniverse from props and then use it.

Reply:
Done